base/package.mustache

```diff
@@ -16,10 +16,12 @@
     "json-schema-deref-sync": "^0.2.8",
     "jsonwebtoken": "^5.4.1",
     "lodash": "^3.10.1",
+    "password": "https://github.com/Prismatik/password.git",
```

Comment:
You'll need the `git` prefix as well: `git+https://github.com/Prismatik/password.git`